src/CloudService/custom/Switch-AzCloudService.ps1

```diff
@@ -96,7 +96,7 @@ function Switch-AzCloudService {
 
         # Check that both have swappable property set.
         if ([string]::IsNullOrEmpty($SourceCloudService.NetworkProfile.SwappableCloudService.Id)) {
-            throw "SwappableCloudServiceId is not set on the source cloud service " + $SourceCloudService.Name 
+            throw "SwappableCloudServiceId is not set on the source cloud service ($SourceCloudService.Name)"

```

Comment:
As I tried, variable reference should be $($SourceCloudService.Name). Have you tried it?